### DISCUSSION TITLE
[Feature Request] Add disorder, fake and other fragment settings

### DISCUSSION THREAD

**Discussion by romkazor** [XTLS/Xray-core#3760] [Ideas]
Hello. Please add additional mechanisms to override dpi like disorder, fake and others like in goodbyedpi/byedpi projects because fragment settings are not enough in many cases. Thanks.

**Comment by mmmray** [comment]
I don't think many xray devs are familiar with goodbyedpi since the tool does not appear to work properly in China. That said, maybe some techniques work when applied to xray configs. Somebody has to do the work. As with all things, these functions have to be discussed on an individual basis, and PRs are welcome.
